src/index.ts

```diff
@@ -1267,14 +1276,26 @@ class Piscina extends EventEmitterAsyncResource {
   }
 
   get waitTime () : any {
+    if (!this.#pool.waitTime) {
+      throw Errors.TimeRecordingDisabled();
```

Comment:
Let's not `throw` but return `null` instead.

In that way, we can set the signature of the getter to return either `HistogramSummary | null`

Reply:
Since the type is already `any`, it could definitely return null instead of throwing.  Changing the type would be a breaking change though. All consumers, regardless of the new option's value, would need to add a guard before using the result with the more strict, although more accurate, return type.
Maybe the return type change would be a good v5 candidate as well?

Reply:
Hmm, good point, that SGTM, let's keep it as is, and just return `null` instead, thanks for pointing that out!